### DISCUSSION TITLE
Prompt Before Deletion

### DISCUSSION THREAD

**Discussion by mricher-git** [AssetRipper/AssetRipper#737] [General]
I exported a complete game to a folder, spend hours cleaning it up. Went back to AR to extract a single selected asset and it nuked my entire working directory. Seriously?

if (Directory.Exists(path))
await Task.Run(() => Directory.Delete(path, true));

Unreal, looks like it could have been worse, don't choose the wrong directory ffs. Imagine I chose to dump the file on my root.

**Comment by ds5678** [comment]
#738
